src/pages/jeonghyunKim/Main/FeedComponent.js

```diff
@@ -0,0 +1,341 @@
+  const [viewAllCommentFlag, setViewAllCommentFlag] = useState(false);
+
+  const feedImgDotContainerRef = useRef();
+
+  const feedImgUlRef = useRef();
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/commentData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setCommentData(data);
+      });
+  }, []);
+
+  useEffect(() => {
+    const feedContentsReduce = async () => {
+      const feedContentsCommentBody = document.getElementsByName(
+        'feedContentsCommentBody' + feedItem.id
+      )[0];
+
+      let temp = [];
+
+      const feedLoopWork = () => {
+        temp.push(feedContentsCommentBody.innerHTML);
+        const reduceCommentString =
+          feedContentsCommentBody.innerHTML.slice(0, 10).trim() + '...';
+        feedContentsCommentBody.innerHTML = reduceCommentString;
+      };
+
+      const setFeedOriginState = () => {
+        setFeedOriginContent(temp[0]);
+      };
+
+      await feedLoopWork();
+      await setFeedOriginState();
+    };
+    feedContentsReduce();
+  }, [feedItem.id]);
+
+  useEffect(() => {
+    for (let i = 0; i < feedImgDotContainerRef.current.children.length; i++) {
+      if (feedImgDotPosition === i) {
+        feedImgDotContainerRef.current.children[i].children[0].style.color =
+          'rgba(var(--d69,0,149,246),1)';
+      } else {
+        feedImgDotContainerRef.current.children[i].children[0].style.color =
+          '#adadad';
+      }
+    }
+  }, [feedImgDotPosition]);
+
+  const feedExtendContents = event => {
+    event.preventDefault();
+    const extendBtn = document.getElementsByName(
+      'feedExtendContents' + feedItem.id
+    )[0];
+
+    extendBtn.style.display = 'none';
+    const feedContentsCommentBody = document.getElementsByName(
+      'feedContentsCommentBody' + feedItem.id
+    )[0];
+    feedContentsCommentBody.innerHTML = feedOriginContent;
+  };
+
+  const onChangeFeedComment = event => {
+    event.preventDefault();
+    const { value } = event.target;
+    const inputCommentsSubmitBtn = document.getElementsByName(
+      'inputCommentsSubmitBtn' + feedItem.id
+    )[0];
+
+    if (value.replace(/(^\s*)|(\s*$)/gi, '')) {
+      inputCommentsSubmitBtn.style.opacity = 1;
+      inputCommentsSubmitBtn.style.cursor = 'pointer';
+    } else {
+      inputCommentsSubmitBtn.style.opacity = 0.3;
+      inputCommentsSubmitBtn.style.cursor = 'inherit';
+    }
+  };
+
+  const enterFeedComment = event => {
+    if (event.key === 'Enter') {
+      addFeedComment(event);
+    }
+  };
+
+  const addFeedComment = event => {
+    event.preventDefault();
+    const inputCommentsSubmitBtn = document.getElementsByName(
+      'inputCommentsSubmitBtn' + feedItem.id
+    )[0];
+    const inputCommentsTextArea = document.getElementsByName(
+      'inputCommentsTextarea' + feedItem.id
+    )[0];
+
+    let inputComments = inputCommentsTextArea.value.replace(/\n/g, '');
+
+    if (inputComments.length > 0) {
```

Comment:
```suggestion
    if (inputComments.length) {
```
이렇게 해도 좋을듯합니다~!!! 아시다시피 어차피 `length`가 1개 이상이라면 `true`일테니까요.

Reply:
뭔가 딱딱 떨어지지 않으면 안되는 그런 게 있었어요 ㅠ

수정되었습니다